tests/test_deprecations.py

```diff
@@ -227,33 +227,42 @@ def test_deprecation_shelve_functions():
 
 
 def test_deprecations_shelf_function_results():
-    np.testing.assert_allclose(
-        pfilt.high_shelve(None, 10e3, 1.5, 1, "I",
-                          sampling_rate=44100).coefficients,
-        pfilt.high_shelf(None, 10e3, 1.5, 1, "I",
-                         sampling_rate=44100).coefficients,
-        rtol=0)
-
-    np.testing.assert_allclose(
-        pfilt.low_shelve(None, 10e3, 1.5, 1, "I",
-                         sampling_rate=44100).coefficients,
-        pfilt.low_shelf(None, 10e3, 1.5, 1, "I",
-                        sampling_rate=44100).coefficients,
-        rtol=0)
-
-    np.testing.assert_allclose(
-        pfilt.high_shelve_cascade(None, 250, "lower", -60, None, 4,
-                                  sampling_rate=44100)[0].coefficients,
-        pfilt.high_shelf_cascade(None, 250, "lower", -60, None, 4,
-                                 sampling_rate=44100)[0].coefficients,
-        rtol=0)
-
-    np.testing.assert_allclose(
-        pfilt.low_shelve_cascade(None, 250, "lower", -60, None, 4,
-                                 sampling_rate=44100)[0].coefficients,
-        pfilt.low_shelf_cascade(None, 250, "lower", -60, None, 4,
-                                sampling_rate=44100)[0].coefficients,
-        rtol=0)
+    with pytest.warns(PyfarDeprecationWarning,
+                      match="'high_shelve' will be deprecated in pyfar 0.9.0"):
+        np.testing.assert_allclose(
+            pfilt.high_shelve(None, 10e3, 1.5, 1, "I",
+                            sampling_rate=44100).coefficients,
+            pfilt.high_shelf(None, 10e3, 1.5, 1, "I",
+                            sampling_rate=44100).coefficients,
+            rtol=0)
+
+    with pytest.warns(PyfarDeprecationWarning,
+                      match="'low_shelve' will be deprecated in pyfar 0.9.0"):
+        np.testing.assert_allclose(
+            pfilt.low_shelve(None, 10e3, 1.5, 1, "I",
+                            sampling_rate=44100).coefficients,
+            pfilt.low_shelf(None, 10e3, 1.5, 1, "I",
+                            sampling_rate=44100).coefficients,
+            rtol=0)
```

Comment:
```suggestion
        np.testing.assert_allclose(
            array([[[ 1.08735675,  1.10156456,  0.01420782],
                    [ 1.        ,  0.92685107, -0.07314893]]]),
            pfilt.low_shelf(None, 10e3, 1.5, 1, "I",
                            sampling_rate=44100).coefficients,
            rtol=0)
```

I would suggest to manually add the results from the privous implementation as knows truth values. At some point these need to be removed, and the test need to be changed anyways.